cmake/external/grpc.cmake

```diff
@@ -33,7 +33,7 @@ ExternalProject_Add(
     extern_grpc
     DEPENDS protobuf zlib
     GIT_REPOSITORY "https://github.com/grpc/grpc.git"
-    GIT_TAG "v1.10.x"
+    GIT_TAG "v1.8.x"
```

Comment:
What's the difference between them?

Reply:
Done. 1.10 hang on some machine...reverted to original one for now.